Hudl.Mjolnir/Command/Command.cs

```diff
@@ -343,10 +360,16 @@ private Task<TResult> ExecuteInIsolation(CancellationToken cancellationToken)
 
             var workItem = ThreadPool.Enqueue(() =>
             {
+               
+                if (TimeoutsIgnored)
+                {
+                    return UseCircuitBreakers.Value
+                        ? ExecuteWithBreaker(CancellationToken.None)
+                        : ExecuteAsync(CancellationToken.None);
+                }
```

Comment:
This bit could probably be DRYed up a bit with the duplicate block below. Maybe just assign a local token conditionally and use that.